llvm/lib/Target/X86/X86InstrInfo.h

```diff
@@ -29,8 +29,10 @@ class X86Subtarget;
 namespace X86 {
 
 enum AsmComments {
+  // For instr that was compressed from EVEX to LEGACY.
+  AC_EVEX_2_LEGACY = MachineInstr::TAsmComments,
```

Comment:
Why we need to associate with this value? Will it be a problem if there's new Flag added after `MachineInstr::TAsmComments`?

Reply:
```
  enum CommentFlag {
    ReloadReuse = 0x1,    // higher bits are reserved for target dep comments.
    NoSchedComment = 0x2,
    TAsmComments = 0x4    // Target Asm comments should start from this value.
  };
```

It's documented at llvm/include/llvm/CodeGen/MachineInstr.h. `T` is short for target and `TAsmComments` should always be the last value here.